test/lib/template-registry-helper.test.js

```diff
@@ -10,8 +10,9 @@
  * governing permissions and limitations under the License.
  */
 
-const { getTemplates } = require('../../src/lib/template-registry-helper')
+const { getTemplates, /*addTemplate, removeTemplate*/ } = require('../../src/lib/template-registry-helper')
 const templateRegistrySDK = require('@adobe/aio-lib-templates')
+// const { when } = require('jest-when')
```

Comment:
Let's have the implementation part in this PR only. As we agreed tests will be added in a separate ticket.